source/mbed_trace.c

```diff
@@ -20,7 +20,7 @@
 #ifdef MBED_CONF_MBED_TRACE_ENABLE
 #undef MBED_CONF_MBED_TRACE_ENABLE
 #endif
-#define MBED_CONF_MBED_TRACE_ENABLE 1
+
```

Comment:
This change should be completely unnecessary, regardless of whether compiling with or without tracing enabled.